WoWsShipBuilder.Web/WoWsShipBuilder.Web.csproj

```diff
@@ -20,4 +20,8 @@
       <PackageReference Include="Nerdbank.GitVersioning" PrivateAssets="all" Version="3.6.132" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
Remove this unnecessary block. Content in wwwroot is automatically included by the SDK